src/routes/status.route.ts

```diff
@@ -0,0 +1,32 @@
+import { FastifyInstance } from 'fastify';
+import mongoose from 'mongoose';
+import { connectDB } from '../plugins/db';
+
+export async function statusRoute(app: FastifyInstance) {
+  app.get('/status', async (request, reply) => {
```

Comment:
I am adding a status route for simple verification that the server is running and is connected to the database